### PR TITLE
[HYD-148] Rename to pg_catalog.alter_table_set_access_method

### PR DESCRIPTION
**Pull request by owenthereal**
This makes `alter_table_set_access_method` usability at the root namespace and consistent with the rest of the columnar functions. Tested the following working:

```sql
CREATE TABLE my_table (i INT8);

SELECT alter_table_set_access_method('my_table', 'columnar');
```

### CODE REVIEW

File: src/backend/columnar/sql/downgrades/columnar--11.1-3--11.1-2.sql [hunk 1]
```diff
@@ -0,0 +1,211 @@
+-- columnar--11.1-3--11.1-2.sql
+
+DROP FUNCTION pg_catalog.alter_table_set_access_method;
```
Comment (owenthereal) [inline src/backend/columnar/sql/downgrades/columnar--11.1-3--11.1-2.sql:3]: I tried `ALTER FUNCTION columnar.alter_table_set_access_method RENAME TO pg_catalog.alter_table_set_access_method` and pg doesn't allow me to rename a function with the `.` prefix

**Review by wuputah — CHANGES_REQUESTED** [review]
Apologize for the miscommunication in the issue text, but my opinion is we want to go the other direction: all functions from this extension should live in `columnar` (or even `columnar_internal` which is being used for some things upstream now). If we want to put "common" things into `pg_catalog`, I think they could just be wrappers, but I'm not sure any of the functions rise to that level.

**Comment by owenthereal** [comment]
Okay, there are two other functions that are at the `pg_catalog` level: `alter_columnar_table_set` & `alter_columnar_table_reset`. I will revise this PR and add the `columnar` prefix to them to be consistent with `alter_table_set_access_method` then

**Comment by owenthereal** [comment]
Closing. We will add `columnar.` prefix to the rest of `alter_*` functions instead.

**Pull request closed without merging**